src/android/InAppBrowser.java

```diff
@@ -6,9 +6,7 @@ Licensed to the Apache Software Foundation (ASF) under one
        to you under the Apache License, Version 2.0 (the
        "License"); you may not use this file except in compliance
        with the License.  You may obtain a copy of the License at
-
          http://www.apache.org/licenses/LICENSE-2.0
-
        Unless required by applicable law or agreed to in writing,
```

Comment:
The deletion of these lines will probably cause problems with our release process where it checks for proper license headers. Please revert these deletions